onboarding/README.md

```diff
@@ -9,5 +9,6 @@ Welcome to Artsy! We're excited to have you.
 | [For New Hires](/onboarding/new-hires.md) | What should I do in my first few weeks at Artsy? |
 | [For Managers of New Hires](/onboarding/managers.md) | What should I remember to do for new hires? |
 | [For Mentors](/onboarding/mentors.md) | How do I be the most effective mentor I can be? |
+| [Sprint Rotation](/onboarding/sprint-rotation.md) | What is the sprint rotation and how does it work? |
```

Comment:
The new-hires and managers docs linked above should probably also mention [or link to] the sprint rotation themselves.